airsonic-main/src/main/java/org/airsonic/player/service/search/SearchServiceUtilities.java

```diff
@@ -71,6 +79,45 @@
     @Autowired
     private MediaFileService mediaFileService;
 
+    private static final Logger LOG = LoggerFactory.getLogger(SearchServiceUtilities.class);
+
+    private Random random;
+    
+    private static Random createSecureRandom(String algorithm) {
+        SecureRandom secureRandom = null;
+        try {
+            LOG.debug("{} is used to create a random list of songs.", algorithm);
+            secureRandom = SecureRandom.getInstance(algorithm);
```

Comment:
Why can't we just do new `SecureRandom()` and call it a day?

Reply:
Regardless of the VM implementation, it is clearly stated that only these two algorithms will be used.
SecureRandom() gives almost the same result, but it depends on the environment setup and VM implementation.

Every environment is likely to hold one of the two algorithms listed here.
And these are algorithms that do not block the system.